nextflow.config

```
@@ -8,26 +8,94 @@
 // Global default params, used in configs
 params {
 
+  // Nf-core lint required params that are unused
+  input = 'foobarbaz'
```

If you set this to a string like this, the "required" behaviour won't work because the value will always be set by default. Setting it to `null` should do what you want I think. Trying to run the pipeline with it being set will fail though as it's set to required in the schema.

```suggestion
  input = null
```